cmd/kube-controller-manager/app/options/options.go

```diff
@@ -46,6 +46,7 @@ func NewCMServer() *CMServer {
 			Port:                              ports.ControllerManagerPort,
 			Address:                           "0.0.0.0",
 			ConcurrentEndpointSyncs:           5,
+			ConcurrentServiceSyncs:            5,
```

Comment:
why do we need more than 1? im guessing the  bottleneck here is cloud api. if it doesnt result in e2e failure, suggest just setting it to 1.